### ISSUE TITLE
Settlers avoid settling near Natural Wonders

### ISSUE THREAD

**Issue by adaneslavo** [LoneGazebo/Community-Patch-DLL#9284]
I gave both civs 5 settlers and no one settled near NW. They have special yields (science, culture and other), so they should be proritized.
![obraz](https://user-images.githubusercontent.com/33016300/195371926-8da649df-6e3e-4c0b-9bae-7e3281cb5fd4.png)
![obraz](https://user-images.githubusercontent.com/33016300/195371970-5c220c20-87de-4566-a692-47ad1012d0f6.png)

**Comment by adaneslavo** [comment]
![obraz](https://user-images.githubusercontent.com/33016300/195373696-7b6956c9-f37e-441c-903d-dfbe3e9ce84f.png)
![obraz](https://user-images.githubusercontent.com/33016300/195373863-062e06a5-22d7-4d23-b40c-1ca97e9ead95.png)

**Comment by ilteroi** [comment]
all corner cases in bad terrain or too far out except for liegnitz. and even there the better spot is probably just too far out of their comfort zone.

there are many tradeoffs here and i'm quite happy with the current state ... you can go play with the parameters in the database but i would not touch the algorithm.

**Comment by adaneslavo** [comment]
Which parameters do you mean?

**Comment by adaneslavo** [comment]
Egypt settles by himself in much wors place. Also on desert, but without Fish and only one Sugar. Few tiles on the right and he would access to Stone, 2 Fish, NW, and Jade.

**Comment by adaneslavo** [comment]
> except for liegnitz
That's what I cannot understand the most...

**Comment by adaneslavo** [comment]
> even there the better spot is probably just too far out of their comfort zone

what is the comfort zone?

**Comment by Stalker012345** [comment]
Yeah most of those spots look reasonable, as especially with a little border buying the AI can absorb the NW into their territory anyway.

**Comment by LessRekkless** [comment]
![image](https://user-images.githubusercontent.com/34615525/195968955-0bdbc2cd-5cd4-4219-832a-495904dd2519.png)
Salzburg settles right next to Sri Prada here.

**Comment by adaneslavo** [comment]
@ilteroi Where are those parameters you talked about, and what are their names?

**Comment by ilteroi** [comment]
CITY_RING_*_MULTIPLIER detemines how much (relative) weight they put on the plots in each ring
SETTLER_*_MULTIPLIER is how much weight the put on the different yields
BUILD_ON_*_PERCENT some specials
SETTLER_DISTANCE_DROPOFF_MODIFIER how far out from existing cities they like to settle

**Comment by adaneslavo** [comment]
Are those Defines?
